src/components/RandomWorldButton.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {  useNavigate  } from "react-router-dom";
+import { Button } from "@mui/material";
+
+
+const RandomWorldButton = (props) => {
```

Comment:
```suggestion
const RandomWorldButton = ({worlds}) => {
```
Destructuring the props in the function signature helps to improve readability.